os_migrate/plugins/module_utils/workload_common.py

```diff
@@ -61,6 +94,8 @@ def wait_for_lock(self):
                     self.log.debug('Released lock: %s', result)
                 except subprocess.CalledProcessError as err:
                     self.log.error('Error releasing lock: %s', str(err))
+                    check_and_cleanup_lockfiles()
```

Comment:
The lock which is being created by this use_lock decorator is cleaned up on line 93 above. It probably won't hurt but if line 93 didn't succeed, i'm not sure if it's worth trying again.